### PR TITLE
MNT: Sample field is now a string

### PR DESCRIPTION
**Pull request by danielballan**

### CODE REVIEW

File: metadataservice/schema/run_start.json [hunk 1]
```diff
@@ -9,7 +9,7 @@
             "description": "Name of project that this run is part of"
         },
         "sample": {
-            "type": ["object"],
+            "type": ["string"],
```
Comment (tacaswell) [inline metadataservice/schema/run_start.json]: I thought it was string or object?

**Comment by ericdill** [comment]
Why not just use the event-model project?

**Comment by tacaswell** [comment]
inertia, this should probably get added with BS to the list of projects to be refactored.

**Comment by danielballan** [comment]
Gah. I had left a line note to that effect (sample can be object or string) but did not realize it hadn't been addressed when I cherry picked it today. Should have checked.

Sent from Outlook Mobile<https://aka.ms/qtex0l>




On Fri, Apr 1, 2016 at 9:23 AM -0700, "Thomas A Caswell" <notifications@github.com<mailto:notifications@github.com>> wrote:


inertia, this should probably get added with BS to the list of projects to be refactored.

**Pull request merged**